sdks/java/container/build.gradle

```diff
@@ -55,7 +54,7 @@ dependencies {
 }
 
 golang {
-  packagePath = 'github.com/apache/beam/sdks/java/boot'
+  packagePath = 'github.com/apache/beam/sdks/java/container'
```

Comment:
It's astounding things were still working with that directory being wrong?

Reply:
Yeah. I have no idea. Probably has something to do with how gogradle was doing stuff under the hood, if I had to guess.